tests/integration/index.html

```diff
@@ -18,12 +18,6 @@
       var should = chai.should();
       var assert = chai.assert;
     </script>
-    <script src='deps/pouchdb-1.1.0-postfixed.js'></script>
-    <script src='deps/pouchdb-2.0.0-postfixed.js'></script>
-    <script src='deps/pouchdb-2.2.0-postfixed.js'></script>
-    <script src='deps/pouchdb-3.0.6-postfixed.js'></script>
-    <script src='deps/pouchdb-3.2.0-postfixed.js'></script>
-    <script src='deps/pouchdb-3.6.0-postfixed.js'></script>
     <script src='utils-bundle.js'></script>
```

Comment:
I am confused at how we are actually testing these migrations without pulling in the old versions?